roofit/hs3/inc/LinkDef.h

```diff
@@ -0,0 +1,10 @@
+#ifdef __CINT__
+#pragma link off all globals;
+#pragma link off all classes;
+#pragma link off all functions;
+
+#pragma link C++ class RooStats::HistFactory::JSONTool+;
+#pragma link C++ class RooJSONFactoryWSTool+;
+#pragma link C++ class RooJSONFactoryWSTool::Importer+;
+#pragma link C++ class RooJSONFactoryWSTool::Exporter+;
+#endif
```

Comment:
Do you need I/O for these JSON tools?
- Yes: Ok, need to declare them.
- No: Don't create a `LinkDef.h`

Reply:
I don't need I/O, but I was under the impression that a dictionary is needed for python bindings. But perhaps one can also trigger the dictionary generation without a LinkDef?